ConcernsCaseWork/ConcernsCaseWork.API/StartupConfiguration/ApiConfigurationExtensions.cs

```diff
@@ -1,11 +1,26 @@
+using System.Net.Mime;
+
 namespace ConcernsCaseWork.API.StartupConfiguration;
 
 public static class ApiConfigurationExtensions
 {
 	public static IServiceCollection AddApi(this IServiceCollection services, IConfiguration configuration)
 	{
```

Comment:
Not sure why this would have changed

Reply:
I should have mentioned in the comments - I mentioned to Mike in chat. It's refactoring to remove an unused method and tidy up the extension classes - no effective changes, but it is more readable (IMO)